src/datapilot/core/platforms/dbt/wrappers/manifest/v12/schemas.py

```diff
@@ -0,0 +1,45 @@
+from typing import Dict
+from typing import Type
+from typing import Union
+
+from dbt_artifacts_parser.parsers.manifest.manifest_v12 import Exposures
+from dbt_artifacts_parser.parsers.manifest.manifest_v12 import Macros
+from dbt_artifacts_parser.parsers.manifest.manifest_v12 import Nodes
+from dbt_artifacts_parser.parsers.manifest.manifest_v12 import Nodes1
+from dbt_artifacts_parser.parsers.manifest.manifest_v12 import Nodes2
+from dbt_artifacts_parser.parsers.manifest.manifest_v12 import Nodes3
+from dbt_artifacts_parser.parsers.manifest.manifest_v12 import Nodes4
+from dbt_artifacts_parser.parsers.manifest.manifest_v12 import Nodes5
+from dbt_artifacts_parser.parsers.manifest.manifest_v12 import Nodes6
+from dbt_artifacts_parser.parsers.manifest.manifest_v12 import Nodes7
+from dbt_artifacts_parser.parsers.manifest.manifest_v12 import Sources
+
+from datapilot.core.platforms.dbt.constants import GENERIC
+from datapilot.core.platforms.dbt.constants import SINGULAR
+
+ManifestNode = Union[
```

Comment:
RPCNode which was present in v11 isn't available here

Reply:
what is this Nodes1?